### ISSUE TITLE
The synchronization does not work

### ISSUE THREAD

**Issue by Glaived** [Yarillo4/warlock-healthstone-tracker-tweaked#1]
> Sync healthstone status among party / raid members

I am a priest who does not use the addon and I play (in a group) with a demonist who uses the addon. The addon knows I don't have a healthstone, the demonist initiates a trade to give me one, and the addon understands this.

In combat, I use my Healthstone, but the addon (demonist player side) still thinks I have it.

**Comment by Yarillo4** [comment]
Yes don't worry about using the addon if you are not a warlock yourself, this is not required

The way the addon works is: the warlock only sees that you've eaten the healthstone if the message appears in his combat log, If the warlock is far away, or in ghost form when you eat the healthstone, then he won't receive that information, and the addon will think that you still have the healthstone.

This is a shortcoming that is not possible to fix unfortunately. When that happens, the only thing to do is physically ask the warlock to give you the healthstone back

If the opposite thing happens, the warlock is attempting to trade you but you still have a healthstone, don't cancel the trade. You need to accept the trade anyway - since the healthstone is unique, this will create a "trade error" that the addon can see, and the addon will understand that you already have one.

**Comment by Glaived** [comment]
I confirm you the presence of a '' bug ''

I am in priest, alive, next to the demonist alive too, I buff with [Power Word: Fortitude](https://classic.wowhead.com/spell=1245), which increases my max life and leaves me at 80%~/100% HP, I use the [Lesser Healthstone](https://classic.wowhead.com/item=5511), but on the side of the demonist the addon considers that I still have my soulstone 😕

Is there any relation to the fact that we are not in combat?

Does the demonist have to change anything in the configuration of the battle log window?

Perhaps a problem related to the fact that the demonist plays on the French version of the game?

**Comment by Yarillo4** [comment]
The localization (language) isn't the issue, that's for sure. I also play on the french version and so does most of my guild 

Fortitude shouldn't affect things either. The addon isn't tracking your health, when you eat a healthstone it's like a spell being cast and it is shown in the combat log

The warlock shouldn't have to change anything in the combat log.

I have no idea. Perhaps you guys are playing on a private server or something?

**Comment by Glaived** [comment]
> Perhaps you guys are playing on a private server or something?

Nope, Classic Sulfuron
